lametro/views.py

```diff
@@ -734,6 +736,115 @@ class LAMetroArchiveSearch(TemplateView):
 class LAMetroContactView(IndexView):
     template_name = 'lametro/contact.html'
 
+
+class MinutesView(EventsView):
+    template_name = 'lametro/minutes.html'
+
+    def _get_historical_events(self, start_datetime=None, end_datetime=None):
+        csv_events = get_list_from_csv('historical_events.csv')
+
+        events_dicts = [dict(e) for e in csv_events]
```

Comment:
You don't need to cast to a dict here, since `get_list_from_csv` returns a list of ordered dicts. You can just work with `csv_events` directly.